examples/ScenarioExample.m

```diff
@@ -0,0 +1,19 @@
+clear all 
+
+% Disable cast to struct warnings
+w = warning ('off','all');
+
+%% Get configuration
+Config = MonsterConfig(); % Get template config parameters
+Config.Scenario.scenario = '3GPP TR 38.901 UMa' ; %Chose scenario
+
```

Comment:
For instance here, why not just make the necessary changes in a script like this and name it accordingly? For instance change the name of this script to 3GPP TR 38.901 UMa and move whatever magic is happening inside the scenario switch statement, to here

Reply:
Such that a different script is available for each scenario instead of a scenario property? I am not sure I understand. Would you rather split the switch statement into different scripts or a single script that does everything?

Reply:
Yes exactly. One script per scenario. E.g. `3GPP_TR_38.901_UMa.m` which then sets the config directly in the script. Then you'd know and can see exactly what is going on. It's also then usable as a template for any further adjustments.

Reply:
It boils down to this for me:
1. Non-advanced users would benefit from a scenario property as it would hide away any details unnecessary (or at a very low level). This is :+1:  imo.
2. It is not transparent to advanced users what is happening unless they inspect how the scenario is applied. This is :-1:  imo.

Reply:
That seems reasonable to me. I'll fix that.

Reply:
So far 2 2gpp, 2 ieee and a single cell scenario. I'll do the separate scripts, and skip the property.